### ISSUE TITLE
Cannot reproduce documentation examples on Utah

### ISSUE THREAD

**Issue by andycasey** [sdss/sdssdb#45]
I would like Astra to use `sdssdb`. However, I am having a lot of trouble getting anything working with `sdssdb`, either locally or at Utah. Detailed description below.

**Describe the bug**
I will try to reproduce the first line of "getting started" at https://sdssdb.readthedocs.io/en/latest/intro.html:

**To Reproduce**
1. Connect to apogee.sdss.org
2. Following the `sdssdb` installation instructions, type `module load sdssdb`.
3. Run `ipython`
4. Execute: `from sdssdb.peewee.sdss5db.catalogdb import GaiaDR2Source`

````python
(base) andy@hamish:~/astra$ ssh-utah
Last login: Sun Nov  1 17:15:24 2020 from 118.138.241.44
               Center for High Performance Computing
               -------------------------------------
Help: Please send issues or questions to helpdesk@chpc.utah.edu

News: https://www.chpc.utah.edu/news/latest_news/index.php

Tip:  CHPC recommends that you use FastX for your SSH sessions to
      CHPC interactive resources.
      https://www.chpc.utah.edu/documentation/software/fastx2.php
-------------------------------------------------------------------

                             kingspeak
                             ---------
Allocations are available for priority use of the general nodes.
For more information and details on getting an allocation, see the
documentation.
https://www.chpc.utah.edu/userservices/allocations.php
-------------------------------------------------------------------

Currently Loaded Modulefiles:
 1) postgresql/9.6.1         6) dust/v0_1               11) svn/1.9.5               16) idlutils/v5_5_35        21) speclog/trunk           26) apogeereduceplan/trunk  
 2) pbs/slurm                7) sdsstools/0.1.7         12) idl/8.5.1               17) bbspec/trunk            22) platelist/trunk         27) apogee/trunk            
 3) root/chpc                8) sdss_access/1.1.0       13) photolog/dr10           18) elodie/v1_3             23) photoop/v1_12_3         28) mwm/master              
 4) miniconda/3.7.6_apogee   9) sdss_python_module/v2_3 14) twomass/allsky          19) spinspect/trunk         24) idlspec2d/apogee-v5_6_5 
 5) sdss4tools/0.2.7        10) tree/sdsswork           15) usno/vB                 20) specflat/v1_20          25) cfitsio/v3_27           
[u6020307@apogee:~]$ module load sdssdb
[u6020307@apogee:~]$ ipython
Python 3.6.3 (default, Oct 20 2017, 13:08:04) 
Type 'copyright', 'credits' or 'license' for more information
IPython 6.2.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import sdssdb
sdssdb.__version__
In [2]: print(sdssdb.__version__)
None

In [3]: print(sdssdb.__path__)
['/uufs/chpc.utah.edu/common/home/sdss10/software/sdss/github/sdssdb/0.4.5/python/sdssdb']

In [4]: from sdssdb.peewee.sdss5db.catalogdb import GaiaDR2Source
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-4-2012ee19f382> in <module>()
----> 1 from sdssdb.peewee.sdss5db.catalogdb import GaiaDR2Source

/uufs/chpc.utah.edu/common/home/sdss10/software/sdss/github/sdssdb/0.4.5/python/sdssdb/peewee/sdss5db/__init__.py in <module>()
     34 
     35 
---> 36 database = SDSS5dbDatabaseConnection(autoconnect=True)  # noqa

/uufs/chpc.utah.edu/common/home/sdss10/software/sdss/github/sdssdb/0.4.5/python/sdssdb/connection.py in __init__(self, *args, **kwargs)
    345 
    346         PostgresqlDatabase.__init__(self, None, autorollback=autorollback)
--> 347         DatabaseConnection.__init__(self, *args, **kwargs)
    348 
    349     @property

/uufs/chpc.utah.edu/common/home/sdss10/software/sdss/github/sdssdb/0.4.5/python/sdssdb/connection.py in __init__(self, dbname, profile, autoconnect, dbversion)
     81             self.dbname = f'{self.dbname}_{self.dbversion}'
     82 
---> 83         self.set_profile(profile=profile, connect=False)
     84 
     85         if autoconnect and self.dbname:

/uufs/chpc.utah.edu/common/home/sdss10/software/sdss/github/sdssdb/0.4.5/python/sdssdb/connection.py in set_profile(self, profile, connect)
    139                 self.connect()
    140 
--> 141         return self.connected
    142 
    143     @abc.abstractmethod

/uufs/chpc.utah.edu/common/home/sdss10/software/sdss/github/sdssdb/0.4.5/python/sdssdb/connection.py in connected(self)
    351         """Reports whether the connection is active."""
    352 
--> 353         return self.is_connection_usable()
    354 
    355     @property

AttributeError: 'SDSS5dbDatabaseConnection' object has no attribute 'is_connection_usable'
````

@albireox suggested I use `module load miniconda/3.7.7_operations`. If I do that I raise a different error. Steps to reproduce:
1. Log in to apogee.sdss.org.
2. `module load miniconda/3.7.7_operations`
3. Run `ipython`
4. `from sdssdb.peewee.sdss5db.catalogdb import GaiaDR2Source`

Produces the following:
````python
(base) andy@hamish:~/astra$ ssh-utah
Last login: Sun Nov  1 17:15:58 2020 from 118.138.241.44
               Center for High Performance Computing
               -------------------------------------
Help: Please send issues or questions to helpdesk@chpc.utah.edu

News: https://www.chpc.utah.edu/news/latest_news/index.php

Tip:  CHPC recommends that you use FastX for your SSH sessions to
      CHPC interactive resources.
      https://www.chpc.utah.edu/documentation/software/fastx2.php
-------------------------------------------------------------------

                             kingspeak
                             ---------
Allocations are available for priority use of the general nodes.
For more information and details on getting an allocation, see the
documentation.
https://www.chpc.utah.edu/userservices/allocations.php
-------------------------------------------------------------------

dCurrently Loaded Modulefiles:
 1) postgresql/9.6.1         6) dust/v0_1               11) svn/1.9.5               16) idlutils/v5_5_35        21) speclog/trunk           26) apogeereduceplan/trunk  
 2) pbs/slurm                7) sdsstools/0.1.7         12) idl/8.5.1               17) bbspec/trunk            22) platelist/trunk         27) apogee/trunk            
 3) root/chpc                8) sdss_access/1.1.0       13) photolog/dr10           18) elodie/v1_3             23) photoop/v1_12_3         28) mwm/master              
 4) miniconda/3.7.6_apogee   9) sdss_python_module/v2_3 14) twomass/allsky          19) spinspect/trunk         24) idlspec2d/apogee-v5_6_5 
 5) sdss4tools/0.2.7        10) tree/sdsswork           15) usno/vB                 20) specflat/v1_20          25) cfitsio/v3_27           
[u6020307@apogee:~]$ module load miniconda/3.7.7_operations                                                                                                                                                                     
[u6020307@apogee:~]$ which python
/uufs/chpc.utah.edu/common/home/sdss50/software/pkg/miniconda/3.7.7_operations/bin/python
[u6020307@apogee:~]$ which ipython
/uufs/chpc.utah.edu/common/home/sdss50/software/pkg/miniconda/3.7.7_operations/bin/ipython
[u6020307@apogee:~]$ ipython
Python 3.7.7 (default, Mar 26 2020, 15:48:22) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.13.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import sdssdb                                                                                                                                                                  

In [2]: print(sdssdb.__version__)                                                                                                                                                      
0.4.6a0

In [3]: print(sdssdb.__path__)                                                                                                                                                         
['/uufs/chpc.utah.edu/common/home/sdss50/software/git/sdss/sdssdb/master/python/sdssdb']

In [4]: from sdssdb.peewee.sdss5db.catalogdb import GaiaDR2Source                                                                                                                      
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-4-2012ee19f382> in <module>
----> 1 from sdssdb.peewee.sdss5db.catalogdb import GaiaDR2Source

ImportError: cannot import name 'GaiaDR2Source' from 'sdssdb.peewee.sdss5db.catalogdb' (/uufs/chpc.utah.edu/common/home/sdss50/software/git/sdss/sdssdb/master/python/sdssdb/peewee/sdss5db/catalogdb.py)
````

**Expected behavior**
I would like to be able to import `sdssdb` and follow the documentation examples, so that I can make sure Astra uses the sdss database in the same way as other tools. 

What am I doing wrong here?

**Additional context**
Readthedocs version is 0.4.6, and the second stack trace is using `sdssdb` version 0.4.6a0.

**Comment by albireox** [comment]
Hi @andycasey,

Thanks for reporting that. Yes, the names of some of the `catalogdb` models has changed over time and it seems the docs are not up to date. You can check this example that I updated recently

https://sdssdb.readthedocs.io/en/latest/example.html

In general, if you replace `GaiaDR2Source` with `Gaia_DR2` it will probably work, but I'll look into reproducing all the examples more carefully and update the page.

**Comment by albireox** [comment]
This should be fixed in ce79de552720508aede8b94026fc6068e458508b. I've updated some of the example in ``intro.rst``.

https://sdssdb.readthedocs.io/en/latest/intro.html
